### ISSUE TITLE
Last seconds, if click space results not shown.

### ISSUE THREAD

**Issue by bilimtekno** [anschwa/typing-test#3]
In the last second if you click space button. Results not shown 
![ssss](https://user-images.githubusercontent.com/37639690/62452178-57e38d80-b778-11e9-95d7-0565c755c661.JPG)

**Comment by anschwa** [comment]
Another good bug, thanks!

**Comment by anschwa** [comment]
@bilimtekno You should be able to get the score after typing any other character in the box.

**Comment by bilimtekno** [comment]
@anschwa yes you are right but when you try to write faster you are finishing text via space. And it's not show otomatically. I thnik that's a bug.
